docs/source/install/index.rst

```diff
@@ -3,15 +3,15 @@ Installation
 
 For a quick evaluation of |st2| and package-based installation, start from
 :doc:`Overview <./overview>`, and proceed with the guide for your Linux distribution.
-
+Please note that only 64 bit architecture is currently supported.
 
 .. rubric:: Installations
 
 .. toctree::
     :maxdepth: 1
 
     Reference Deployment <overview>
-    Ubuntu / Debian <deb>
+    Ubuntu / Debian 14.04 (Trusty) <deb>
```

Comment:
There is also 16.04, thanks to @enykeev efforts.
So I think it's better how it is.

Apart of that, 
https://docs.stackstorm.com/install/deb.html#supported-versions
> We support Ubuntu 14.04, and test on Ubuntu Server 14.04 LTS (HVM) Amazon AWS AMI and puppetlabs/ubuntu-14.04-64-nocm Vagrant box. Other Debian based distributions and versions will likely work with some tweaks. You are welcome to try - please report success to the community.

So that's really enough.
Repeating one thing 3 times for people who can't hear won't help.